Sources/Client/Client.cpp

```diff
@@ -63,10 +63,13 @@ DEFINE_SPADES_SETTING(cg_playerMessages, "1");
 DEFINE_SPADES_SETTING(cg_smallFont, "0");
 
 SPADES_SETTING(cg_playerName);
+DEFINE_SPADES_SETTING(dd_specWallhack);
 
 namespace spades {
 	namespace client {
 
+		Client* Client::globalInstance = nullptr;
```

Comment:
global instance of client could very well be insecure. Its used in AreCheatsEnables (BTW HORRIBLE name. Like for real) and it is MUCH better to pass the client instance as pointer to the function. Global variables should be generally avoided as much as possible